geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/iterators/KryoVisibilityRowEncoder.scala

```diff
@@ -68,22 +69,31 @@ class KryoVisibilityRowEncoder extends RowEncodingIterator {
     * @return
     */
   private def encodeV3(keys: java.util.List[Key], values: java.util.List[Value]): Value = {
-    // 1 byte for version + 2 bytes for count + 1 byte for size + 4 bytes per attribute for offsets
-    // + 4 bytes for user data offset + null bit set bytes
-    var length = (count + 2 + IntBitSet.size(count)) * 4
 
-    var valueCursor = length // tracks our output position for copying bytes
+    // Calculate length of serialised attributes, excluding attribute values themselves
+    var length = 1 +            // version
```

Comment:
does this length end up the same, or was it not calculated correctly before?

Reply:
It ends up precisely the same as before, but I thought this was a clearer way to explain it